### DISCUSSION TITLE
EMI Craftables viewer support

### DISCUSSION THREAD

**Discussion by Roadhog360** [vectorwing/FarmersDelight#930] [Ideas]
Continuing from here: https://github.com/emilyploszaj/emi/issues/618
Please add support for EMI's craftables view. More information over there.

**Comment by Roadhog360** [comment]
I should probably note that I really do believe still that this issue should be fixed on their end.

If you would kindly, I hope you can convince the developer that they shouldn't have to make you use use an API for information the game already provides, and it should just take the craftable information that's already from the recipe book by default.

**Comment by cassiancc** [comment]
This has been implemented in https://github.com/vectorwing/FarmersDelight/pull/972 and https://github.com/vectorwing/FarmersDelight/pull/971
